src/AppBundle/Entity/RepositoryStar.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Rhumsaa\Uuid\Uuid;
+
+/**
+ * RepositoryStar.
+ *
+ * @ORM\Table(uniqueConstraints={@ORM\UniqueConstraint(name="repository_star_unique", columns={"user_id", "repository_id"})})
+ * @ORM\Entity(repositoryClass="RepositoryStarRepository")
+ * @ORM\EntityListeners({"RepositoryStarListener"})
+ */
+class RepositoryStar
+{
+    /**
+     * @var string
+     *
+     * @ORM\Id
+     * @ORM\Column(name="uuid", type="string", length=255)
+     */
+    private $uuid;
```

Comment:
Did you try with a composed key on user & repository, and by removing the uuid ?